### ISSUE TITLE
Future enhancements 

### ISSUE THREAD

**Issue by jwdink** [jwdink/eyetrackingR#30]
To avoid cluttering up issues forum, will put brainstorming for all future enhancements here.

**Comment by jwdink** [comment]
Model-based estimates of confidence intervals for when proportion looking passes a threshold (or difference between proportion-looking across within-subjects conditions passes a significance threshold).

**Comment by jwdink** [comment]
Better tools for analyzing "first looks" (e.g., first saccade to AOI, starting from fixation).

**Comment by jwdink** [comment]
Arbitrary DV in "make_time_sequence_data", "make_time_window_data", etc.

**Comment by jwdink** [comment]
Refactor disgusting unmaintainable analyze-time-bins code

**Comment by brockf** [comment]
Add Bayesian time series analysis using brms() and growth curves.

**Comment by jwdink** [comment]
Doesn't this break our workflow? We don't have any functions that internally use lmer, would we be adding a function with brms? Or did you just mean have a vignette with brms? @brockf

**Comment by brockf** [comment]
True: Let's just add a vignette with BRMS using time_sequence data. The only brms-specific benefit is that we can get interpretable times of divergence (based on some threshold of distribution overlap), but this can be discussed in a vignette.

**Comment by samhforbes** [comment]
I actually just came on here to suggest exactly this (doing this at the moment). Divergence estimates using Bayesian analysis would give a more accessible output, and using brms would mean there is no need to change the syntax from your previous vignettes.

**Comment by jwdink** [comment]
@brockf , @sforbes88 Is the idea here to fit a bayesian growth-curve then use the posterior to estimate divergence?

**Comment by samhforbes** [comment]
Yeah, that's what I was thinking, because it's fairly easy to get prediction intervals from brms. Either way, great package guys.
